webviz_subsurface/_providers/ensemble_surface_provider/ensemble_surface_provider_factory.py

```diff
@@ -61,13 +61,19 @@ def instance() -> "EnsembleSurfaceProviderFactory":
         return factory
 
     def create_from_ensemble_surface_files(
-        self, ens_path: str, attribute_filter: List[str] = None
+        self,
+        ens_path: str,
+        attribute_filter: List[str] = None,
+        rel_surface_folder: str = "share/results/maps",
```

Comment:
```suggestion
        rel_surface_folder: str,
```
Should we remove the default value here in order to only have the default path value defined in one place (less risk of later introduce inconsistencies)?

Reply:
I agree :slightly_smiling_face: